pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/ClinicalKitsRoute.java

```diff
@@ -70,9 +72,9 @@ private ClinicalKitDto getClinicalKit(String kitLabel, Optional<BSPKitDto> optio
             clinicalKit.setCollectionDate(kitInfo.getCollectionDate());
             clinicalKit.setSampleCollection(ClinicalKitDao.PECGS);
             DDPInstance ddpInstance = DDPInstance.getDDPInstance(kitInfo.getRealm());
-            clinicalKit.setNecessaryParticipantDataToClinicalKit(optionalBSPKitDto.get().getDdpParticipantId(), ddpInstance);
+            clinicalKit.setNecessaryParticipantDataToClinicalKit(ddpParticipantId, ddpInstance);
+            new ClinicalKitDao().ifTissueAccessionedTriggerDDP(ddpParticipantId, ddpInstance);
```

Comment:
It looks like `ClinicalKitDao` has no state.  If that's the case, then the methods should probably all be static.

Reply:
:doh: I agree! not sure what I did here 😱